db/migrate/20211014131425_add_user_to_task.rb

```diff
@@ -0,0 +1,9 @@
+class AddUserToTask < ActiveRecord::Migration[6.1]
+  def down
+    change_table :tasks do |table|
+      table.integer :user_id, null: true
```

Comment:
It should be a reference instead of just a number. See other migration files to see how relations are implemented.